public/js/index.js

```diff
@@ -119,13 +125,15 @@ function initCommentForm() {
             $(this).removeClass('checked');
             $(this).find('.octicon').removeClass('octicon-check');
             if (hasLabelUpdateAction) {
-                updateIssueMeta($labelMenu.data('update-url'), "detach", $(this).data('id'));
+                updateIssuesMeta($labelMenu.data('update-url'), "detach",
+                    $labelMenu.data('issue-id'), $(this).data('id'));
```

Comment:
I think, or keep function call on a single line, or keep each param in a line, like this:

```js
updateIssuesMeta(
  $labelMenu.data('update-url'),
  "detach",
  $labelMenu.data('issue-id'),
  $(this).data('id')
);
```

Same for similar cases below.